packages/lib/src/Base.svelte

```diff
@@ -13,9 +14,9 @@
 
   let sesamyDesignTokens = `
     * {
-      --s-main-color: var(--sesamy-main-color, 59 130 246);
-      --s-popular-color: var(--sesamy-popular-color, #ff5557);
-      --s-bg-color: var(--sesamy-bg-color, #F6DFDC);
+      --s-main-color: var(--sesamy-main-color, ${hexToHsl('#7D68F4')});
+      --s-bg-color: var(--sesamy-bg-color, ${hexToHsl('#F6DFDC')});
+      --s-bg-opacity: var(--sesamy-bg-opacity, unset);
```

Comment:
Falling back to `unset` we can have conditional logic in our CSS. From `PaywallRenderer.svelte` (background gradient):

```
bg-gradient-to-b
from-[hsla(var(--s-bg-color),var(--s-bg-opacity,1))]
to-[hsla(var(--s-bg-color),var(--s-bg-opacity,0.5))]
```
This takes whatever `--sesamy-bg-color` is set (as `hsl`) and transitions _that color_ from 100% down to 50%. By setting `--sesamy-bg-opacity: 0;` our clients can bypass this entirely and make it so that the background isn't visible, which is a realistic scenario (our paywall without a background at all).

Reply:
This was heavily refactored in:
https://github.com/sesamyab/sesamy-components/pull/56

This is probably a better solution.